redis/src/commands/mod.rs

```diff
@@ -87,11 +87,21 @@ implement_commands! {
         cmd("SET").arg(key).arg(value)
     }
 
+<<<<<<< HEAD
```

Comment:
Can we fix the conflict?

And maybe use active instead of passive voice in the deprecation note :smile: ?